NEWS.md

```
@@ -6,7 +6,14 @@ ClimaCoupler.jl Release Notes
 
 ### ClimaCoupler features
 
-### Split `setup_and_run` in multiple functions. PR[#1251](https://github.com/CliMA/ClimaCoupler.jl/pull/1251)
+#### Switch to `PartitionStateFluxes` by default. PR[#1117](https://github.com/CliMA/ClimaCoupler.jl/pull/1117)
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)